bot.py

```diff
@@ -237,6 +242,7 @@ async def on_member_join(member):
     if member.bot == False:
         await member.guild.get_channel(739647916905332846).send(member.mention + " has joined.\n" + member.guild.get_role(736316470098657342).mention)
         await member.add_roles(member.guild.get_role(743206825176072345), member.guild.get_role(743206597387485324))
+        bot.memberCount+=1
```

Comment:
mite want another one of these for on_member_leave, tho u could remove it if u ignore the not counting bots part as mentioned above

Reply:
i’ll fix that